scripts/data/experiment.json

```diff
@@ -28,6 +28,53 @@
             {"type": "audio/ogg", "src": "horse.ogg"}
         ]
     }, {
+        "id": "one-randomizer",
+        "type": "random-choice",
+        "options": [{
+            "id": "maybe-consent",
+            "type": "consent",
+            "title": "Just checking",
+            "body": "Are you sure you know what you're getting into?",
+            "consentLabel": "I was told there were cookies"
+        }, {
+            "id": "maybe-soundcheck",
```

Comment:
> It might make sense for each of the "options" items to be arrays. This would allow for 0-n followup frames in each branch rather than just a single frame.

We had originally discussed no dependence on previous frames.

Is this a changing requirement based on the customer meeting early today? If we want composable randomized experiments, we can discuss some broader data structure changes.